src/locales/en/translation.json

```
@@ -240,6 +243,7 @@
   "users": "Users",
   "usersCount": "Users count",
   "view": "View",
+  "viewApiDocumentation": "View documentation",
```

maybe:
```suggestion
  "viewApiDocumentation": "View API documentation",
```
if key is "viewApiDocumentation"